test/sql/append.sql

```diff
@@ -0,0 +1,110 @@
+\ir include/create_single_db.sql
+
+-- create a now() function for repeatable testing that always returns
```

Comment:
I think all of this should go into sql_query.sql since you are testing that your optimizations produce same results but different plans. You can add SET timescaledb.constraint_aware_append = false; to sql_query_results_unoptimized.sql. The problem with this as it is now is that you don't have something like sql_query_results_x_diff.out to make sure that the query results are the same between optimized and unoptimized queries.